airbyte-integrations/connectors/source-tiktok-marketing/metadata.yaml

```diff
@@ -11,7 +11,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 4bfac00d-ce15-44ff-95b9-9e3c3e8fbd35
-  dockerImageTag: 2.0.6
+  dockerImageTag: 3.0.1
```

Comment:
I'm bumping the version here to trigger the tests on this source and eventually publish on merge.